crates/pallet-object-store/tests/integration/main.rs

```diff
@@ -0,0 +1,22 @@
+mod mock;
+
+use frame_support::assert_ok;
+use mock::{new_test_ext, Event, ObjectStore, Origin, System, Test};
+use subspace_core_primitives::crypto;
+
+const ACCOUNT_ID: u64 = 100;
+
+#[test]
+fn can_do_put() {
```

Comment:
Ok, my last attempt on this topic. Not a fan of this `integration` way because it's really an API test(unit test) not a _traditional_ integration test. This way does not gain more benefits from the API testing perspective. We can build an integration test system properly once we really need it and have the manpower to take care of it.

Reply:
It is integration from the point of view of Rust test system, only those that are in the `src` are considered unit tests. I can rename it if you'd like. But the primary point is to make sure we only access public API as anyone else would, not implementation details. This makes sure we re-export all the required stuff and such.

Reply:
Yeah, I can understand that and that's fair in that aspect. We're certainly a Rust project, but more specifically, we are also a blockchain project that is built with Rust, so it makes more sense to me to use the concepts that are more common in the existing blockchain projects. So I really suggest that we treat them as unit tests.

Reply:
Moved back into `src`